### DISCUSSION TITLE
Calculating Performance of Validators for Condor

### DISCUSSION THREAD

**Discussion by msanlisavas** [casper-network/condor-info#36] [Q&A] [answered]
How will the condor affect the way we calculate the performance of validators? Would love to get an insight for this so that I can implement the solution.
There is a brief example of how it is calculated right now which was directly asked by Ed.
https://eraguardian.com/mainnet/trend

**Comment by melpadden** [comment]
There is a technical article in the works which deals with the impact of the Condor changes on the tokenomics of the chain. 
The (really) low-res view is that on average, rewards will stay the same, and volatility of rewards will have less volatility for smaller validators. This is because every finality signature is now counted. 

This is obviously an important topic, so we will be publishing a much more in-depth view of the new rewards model very soon. We will put a link to this article here.

**Comment by sacherjj** [comment, marked as answer]
There are changes in how rewards and participation is tracked internally to remove some of the issues which existed prior to Condor.  However, the calculation of how a node performed will be the same rewards based calculations used prior to Condor.
